misc/userscripts/qute-keepassxc

```diff
@@ -295,20 +297,37 @@ def connect_to_keepassxc(args):
     return kp
 
 
-def select_account(creds):
+def select_account(creds, dmenu):
     try:
         if len(creds) == 1:
             return creds[0]
-        idx = subprocess.check_output(
-                ['rofi', '-dmenu', '-format', 'i', '-matching', 'fuzzy',
-                '-p', 'Search',
-                '-mesg', '<b>qute-keepassxc</b>: select an account, please!'],
-                input=b"\n".join(c['login'].encode('utf-8') for c in creds)
-        )
-        idx = int(idx)
-        if idx < 0:
-            return None
-        return creds[idx]
+        dispstr = ["["+c["group"]+"] "+c["name"]+"("+c["login"]+")" for c in creds]
+        dispstr = "\n".join(dispstr).encode("utf-8")
+        if dmenu=="rofi":
+            idx = subprocess.check_output(
+                    ['rofi', '-dmenu', '-format', 'i', '-matching', 'fuzzy',
+                    '-p', 'Search',
+                    '-mesg', '<b>qute-keepassxc</b>: select an account, please!'],
+                    input=dispstr
+            )
+            idx = int(idx)
+            if idx < 0:
+                return None
+            return creds[idx]
+        elif dmenu=="fuzzel":
+            idx = subprocess.check_output(
+                    ['fuzzel', '--dmenu',
+                     '--index',
+                     '--lines='+str(len(creds)),
+                     '--prompt', 'Select an account:'],
+                    input=dispstr
+            )
+            idx = int(idx)
+            if idx < 0:
+                return None
+            return creds[idx]
+        else:
+            raise Exception(f"Not supported dmenu software {dmenu}")
```

Comment:
I think it makes sense to allow this to take a command that takes the names as args. For example: if someone used `wofi` instead of fuzzel they would need to extend this if statement.